airflow/providers/microsoft/azure/example_dags/example_service_bus_queue.py

```diff
@@ -0,0 +1,105 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.providers.microsoft.azure.operators.azure_service_bus_queue import (
+    AzureServiceBusCreateQueueOperator,
+    AzureServiceBusDeleteQueueOperator,
+    AzureServiceBusReceiveMessageOperator,
+    AzureServiceBusSendMessageOperator,
+)
+
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+
+default_args = {
+    "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+    "azure_service_bus_conn_id": "azure_service_bus_default",
+}
+
+CLIENT_ID = os.getenv("CLIENT_ID", "")
+QUEUE_NAME = "sb_mgmt_queue_test"
+MESSAGE = "Test Message"
+MESSAGE_LIST = [MESSAGE + " " + str(n) for n in range(0, 10)]
+
+with DAG(
+    dag_id="example_azure_service_bus_queue",
+    start_date=datetime(2021, 8, 13),
+    schedule_interval=None,
+    catchup=False,
+    default_args=default_args,
```

Comment:
```suggestion
    default_args={
        "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
        "azure_service_bus_conn_id": "azure_service_bus_default",
    },
```
There was some work a while back to cleanup the example DAGs and improve readbility. A smart part of that involved moving `default_args` directly to the `DAG()` so you don't have to look back up to the reference its value. Not required by any means, but it could be nice to have the consistency.

Reply:
changed to 

```
dag_id="example_azure_service_bus_queue",
    start_date=datetime(2021, 8, 13),
    schedule_interval=None,
    catchup=False,
    default_args={
        "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
        "azure_service_bus_conn_id": "azure_service_bus_default",
    },
    tags=["example", "Azure service bus Queue"],
```